File: src/api/myhealthcare-app/app/Models/Hospital.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use App\Models\MedicalDepartment;

class Hospital extends Model
{
    use HasFactory;

    protected $fillable = [
        'name',
        'contact',
        'description',
        'address',
        'longitude_coordinate',
        'latitude_coordinate'
    ];

    protected $dates = [
        'created_at',
        'updated_at',
    ];

    public function medical_departments(){
        return $this->hasMany(MedicalDepartment::class);
    }
}

File: src/client/app/src/main/java/com/example/myhealthcareapp/models/Hospital.kt
package com.example.myhealthcareapp.models

data class Hospital (
    val hospitalId : Int,
    val hospitalName : String,
    val hospitalPhoneNumber : String,
    val hospitalAddress : String,
    val description : String,
    val longitudeCoordinate : String,
    val latitudeCoordinate : String
)